packages/components/badge/src/BadgeItem.styles.tsx

```diff
@@ -0,0 +1,31 @@
+import { makeVariants } from '@spark-ui/internal-utils'
+import { cva, type VariantProps } from 'class-variance-authority'
+
+import { DEFAULT_INTENT, DEFAULT_SIZE, DEFAULT_TYPE } from './config'
+
+export const styles = cva(
+  ['inline-flex h-fit', 'empty:p-none', 'text-center font-bold', 'rounded-full outline outline-2'],
+  {
+    variants: {
+      intent: makeVariants<'intent', ['error', 'info']>({
```

Comment:
intents might be:
- primary
- secondary
- success
- alert
- danger
- info
- neutral
- surface
for consistency

Reply:
for deciding the color filling strategy: apply the same as button-filled design with no rules for hover-focus-select status